apps/frontend/src/pages/profile/styles.module.css

```
@@ -0,0 +1,21 @@
+.profile-container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	padding: 26px;
```

should be dividable by 4
```suggestion
	padding: 24px;
```